### PR TITLE
Issue #6207: Add xpath regression test for AbstractClassName

### PR DESCRIPTION
**Pull request by mincong-h**
This PR adds XPath regression test for suppression filter on [AbstractClassName](https://checkstyle.org/config_naming.html#AbstractClassName) check, see #6207.

### CODE REVIEW

File: src/it/resources/org/checkstyle/suppressionxpathfilter/abstractclassname/SuppressionXpathRegressionAbstractClassNameOne.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package org.checkstyle.suppressionxpathfilter.abstractclassname;
+
+public abstract class SuppressionXpathRegressionAbstractClassNameOne { // warn
```
Comment (romani) [inline src/it/resources/org/checkstyle/suppressionxpathfilter/abstractclassname/SuppressionXpathRegressionAbstractClassNameOne.java]: > SuppressionXpathRegressionAbstractClassNameOne 

it is better to make some reasonable names if possible.
For example:
SuppressionXpathRegressionAbstractClassNameTop
SuppressionXpathRegressionAbstractClassNameInner
SuppressionXpathRegressionAbstractClassNameNoModifier

test methods should be named in same patterns:
testModifierOne => testNoModifier

Comment (mincong-h) [reply src/it/resources/org/checkstyle/suppressionxpathfilter/abstractclassname/SuppressionXpathRegressionAbstractClassNameOne.java]: Sure, let me change them.

Comment (rnveach) [reply src/it/resources/org/checkstyle/suppressionxpathfilter/abstractclassname/SuppressionXpathRegressionAbstractClassNameOne.java]: @romani Correct me if I am wrong, but all input files should start with `Input`.
It looks like our test to catch this isn't looking at the IT tier.

Comment (romani) [reply src/it/resources/org/checkstyle/suppressionxpathfilter/abstractclassname/SuppressionXpathRegressionAbstractClassNameOne.java]: lets proceed as we have now, after reply of @timurt , I will make a decision.

**Pull request merged**